plugins/vendor/zoom/ZoomDropFolderPlugin/batch/processors/zoomWebinarProcessor.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @package plugins.ZoomDropFolder
+ */
+
+class zoomWebinarProcessor extends zoomRecordingProcessor
+{
+	
+	protected function getAdditionalUsersFromZoom($recordingId)
+	{
+		return $this->zoomClient->retrieveWebinarPanelists($recordingId);
+	}
+	
+	protected function parseAdditionalUsers($additionalUsersZoomResponse)
+	{
+		$panelists = new kZoomPanelists();
+		$panelists->parseData($additionalUsersZoomResponse);
+		$panelistsEmails = $panelists->getPanelistsEmails();
+		if($panelistsEmails)
+		{
+			KalturaLog::debug('Found the following panelists: ' . implode(", ", $panelistsEmails));
+			$result = array();
+			foreach ($panelistsEmails as $panelistEmail)
+			{
+				$zoomUser = new kZoomUser();
+				$zoomUser->setOriginalName($panelistEmail);
+				$zoomUser->setProcessedName($this->processZoomUserName($panelistEmail));
+				$result[] = $zoomUser;
+			}
+		}
+		else
+		{
+			$result = null;

```

Comment:
No need, empty array is as same as null, just init $result = array(); in the begining